docs/docs/user/projects/models/update.md

```diff
@@ -3,22 +3,28 @@
  ~ SPDX-License-Identifier: Apache-2.0
  -->
 
-# Update a Model to a Newer Version
+# Update a Capella Model to a Newer Version
```

Comment:
As it's tool specific now it belongs to the tool specific docs (tools > capella)